api/src/main/java/ai/djl/training/TrainingConfig.java

```diff
@@ -70,6 +72,13 @@
      */
     Initializer getInitializer();
 
+    /**
```

Comment:
If the initializer requires a Predicate, then initializing the model will require multiple initializers. This function is not defined for multiple initializers.

Reply:
good point, I will extend to a list